modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * @file
+ * The iNope Policy Review Module.
+ */
+
+/**
+ * Implements hook_menu().
+ */
+function inope_policy_review_menu() {
+  $items = array();
+
+  $items['node/%/review'] = array(
+    'title callback' => 'inope_policy_review_tab_title_callback',
+    'title arguments' => array(1),
+    'description' => 'Review of policy nodes.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('inope_policy_review_form'),
```

Comment:
OK - read this:
https://www.drupal.org/node/224170
This talks about wildcard loads.  See here that node/%/review is really a wildcard loader. With wildcard loaders you can have the value of the node passed in via page arguments, there is an example of how that's done.  So essentially you won't need to do a node load on your form. You could use this to set hidden values on your form or add to the form vars.